runtime/doc/quickfix.txt

```diff
@@ -1105,6 +1106,9 @@ conversion can be used to locate lines without a line number in the error
 output.  Like the output of the "grep" shell command.
 When the pattern is present the line number will not be used.
 
+The "%o" conversion will set up module name in quickfix list.  If present it
+will be used in quickfix error window instead of the filename.
+
```

Comment:
You may want to mention that the module name is only used for display purposes
and the file name is used when jumping to the file.